zenoss_hipchat/config.py

```diff
@@ -5,12 +5,12 @@
 import os
 
 
-HIPCHAT_API_V1_TOKEN = os.environ.get('HIPCHAT_TOKEN', None)
+HIPCHAT_API_V2_TOKEN = os.environ.get('HIPCHAT_TOKEN', None)
 HIPCHAT_ROOM_ID = os.environ.get('HIPCHAT_ROOM', None)
-HIPCHAT_API_V1_ENDPOINT = os.environ.get(
+HIPCHAT_API_V2_ENDPOINT = os.environ.get(
     'HIPCHAT_API_ENDPOINT',
     'api.hipchat.com'
 )
-HIPCHAT_FROM = os.environ.get('HIPCHAT_FROM', 'Zenoss')
+HIPCHAT_FROM = os.environ.get('HIPCHAT_FROM', '')
```

Comment:
Why are you changing the default here? Seems to me, that it should stay the same